article-analyzer/src/main/java/co/inventorsoft/academy/homework/nofifier/impl/SlackNotificationServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package co.inventorsoft.academy.homework.nofifier.impl;
+
+import co.inventorsoft.academy.homework.model.Article;
+import co.inventorsoft.academy.homework.model.User;
+import co.inventorsoft.academy.homework.nofifier.NotificationService;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SlackNotificationServiceImpl implements NotificationService {
+    @Override
+    public void notify(User user, Article article) {
+        System.out.printf("Sending Slack notification about article with name %s to user %s...\n", article.getName(), user.getUsername());;
+    }
+
+    @Override
+    public String getSenderIdentifier() {
+        return "SLACK";
```

Comment:
you may return NotificationType.SLACK here, then it will be easier to compare

Reply:
same to email

Reply:
done